### DISCUSSION TITLE
Can't build IPA?

### DISCUSSION THREAD

**Discussion by skuplia** [SoCuul/SCInsta#91] [Support] [answered]
I keep getting an error code 1. 

Creating the IPA file...
[*] extracting ipa..
[!] not a zip/ipa file
[*] deleting temporary directory..
Error: Process completed with exit code 1.

**Comment by 1s0n** [comment]
i have the same issue, its probably from bad ipa file, however i did successfully extract the ipa on my mac so idk what the problem is

**Comment by SoCuul** [comment]
I'd recommend trying to get it from another source in this case, as I'm guessing your ipa is modified in some way.

**Reply by 1s0n** [reply]
Hey I tried getting it from a few other sources but they were all the same file. Are you aware of any websites to download it?

**Reply by SoCuul** [reply, marked as answer]
Actually, if I had to guess upon further inspection I'm guessing your direct download link is incorrect. Try hosting the file in another site to see if that works.

**Reply by 1s0n** [reply]
Yes that was the problem, thank you very much! I hosted file with a http file server on my computer and it worked.

**Reply by 1s0n** [reply]
Should I make a pull request to change it on the wiki?

**Reply by 1s0n** [reply]
nvm i just realised you cant do that with wikis

**Reply by SoCuul** [reply]
Glad that was able to solve your problems!
